doc/source/dev/contributor/devpy_test.rst

```diff
@@ -0,0 +1,145 @@
+:orphan:
+
+.. _devpy-runtest:
+
+===========================
+Running SciPy Tests Locally
+===========================
+
+Basic test writing and execution from within the Python interpreter is
+documented in the `NumPy/SciPy Testing Guidelines`_. This page includes
+information about running tests from the command line using SciPy's
+``dev.py`` command line tool. *Note: Before beginning, ensure that* |pytest|_
+*is installed.*
+
+To run all tests, navigate to the root SciPy directory at the command
+line and execute
+
+::
+
+   python dev.py test -v
```

Comment:
I would start with the default, no `-v`. The `pytest` defaults are pretty sensible, and so are the `dev.py` ones, so no need to consistently add a non-default flag.

Reply:
That makes sense, thanks!